feature_engine/selection/shuffle_features.py

```diff
@@ -0,0 +1,213 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.metrics import roc_auc_score, mean_squared_error, r2_score
+
+from feature_engine.dataframe_checks import _is_dataframe, \
+    _check_input_matches_training_df
+
+from feature_engine.variable_manipulation import _define_variables, \
+    _find_all_variables
+
+
```

Comment:
we follow pep8 for code style.

According to pep8 imports should be first built in, second external libraries, and third local libraries, each block separated by an empty line, and within each block in alphabetical order.

So, sklearn.ensemble before sklearn.metrics, and then sklearn.models

Also, we do not use \, we make imports within parenthesis if more than 1 import from same module. This in particular, would be automatically corrected by black.

Thus:

```
from feature_engine.dataframe_checks import (
   import1,
   import 2,
)
```

Reply:
Which version of black are you using? 
It didn't automatically add parenthesis after import when there are more than one module,  in my case.